Makefile

```diff
@@ -14,9 +14,24 @@ clean: ## delete generated byte code and coverage reports
 	rm -rf assets
 	rm -rf pii_report
 
+define COMMON_CONSTRAINTS_TEMP_COMMENT
+# This is a temporary solution to override the real common_constraints.txt\n# In edx-lint, until the pyjwt constraint in edx-lint has been removed.\n# See BOM-2721 for more details.\n# Below is the copied and edited version of common_constraints\n
```

Comment:
Can you replace `\n` with actual line breaks?

Reply:
It will raise error as we are defining a constant here

Reply:
I'm very confused. You cannot define multiple lines of comments like

```
# line 1
# line 2
# line 3
```
?

Reply:
I am not adding comments here. I am just creating a string constant to be added in common_constraints file. It will be a comment in that file but its just a string constant here

Reply:
Could you add something like the following above the definition?:
```
## The following defines a string constant that contains a multi-line comment.
```
FYI: @jinder1s